src/local.ts

```diff
@@ -0,0 +1,34 @@
+import { handler } from './lambda';
+let AWS = require('aws-sdk');
+
+/**
+ * For testing locally:
+ * `yarn run local <source bucket> <object key>`
```

Comment:
for the future: It would be nice if we had a mocking library so that tests can happen in memory

Reply:
good point. This looks promising https://www.npmjs.com/package/aws-sdk-mock